ssf-1.1.0/README_WOLF.md

```diff
@@ -0,0 +1,114 @@
+# Building SSF with wolfSSL
+
+Port of SSF 1.1.0 with Boost/Asio 1.65.1 for wolfSSL. Project completed 07/2018.
+
+## Overview
+
+The SSF tool uses the Boot.Asio component for the TLS interface. WolfSSL has been ported into Boost.Asio and SSF builds. 
+
+The build option `ASIO_USE_WOLFSSL` is used to indicate wolfSSL as the TLS provider.
+
+## Getting Sources
+
+1. Clone the master version of wolfSSL with 
+
+    $ `git clone https://github.com/wolfSSL/wolfssl.git`
+    OR
+    $ `scp wolfssl-3.15.3.tar.gz ubuntu@204.48.22.68:~/ssfwolfbuild`
+    $ `tar -xzf wolfssl-3.15.3.tar.gz`
+    $ `mv wolfssl-3.15.3 wolfssl`
+
+2. Execute the following commands from the wolfSSL root directory
+
+    $ `cd wolfssl`
+    $ `./autogen.sh` (if git clone)
+    $ `./configure --enable-asio` [--enable-debug]
+    $ `make`
+    $ `make check`
+    $ `sudo make install`
+    $ `sudo ldconfig`
+
+   To Enable Intel Optimizations use:
+   $ `./configure --enable-asio --enable-sp --enable-intelasm --enable-aesni --enable-intelrand`
+
+3. Download SSF 1.1.0 and extract
+
+    https://github.com/securesocketfunneling/ssf/releases
+    $ `wget https://github.com/securesocketfunneling/ssf/archive/1.1.0.tar.gz`
+    $ `tar -xzf 1.1.0.tar.gz`
+    Extracts to ssf-1.1.0
+
+4. Download ASIO 1.65.0 WolfSSL modified
+
+    $ `scp asio_1_65_1.tar.gz ubuntu@204.48.22.68:~/ssfwolfbuild`
+    $ `tar -xzf asio_1_65_1.tar.gz`
+
+5. Download boost 1_65_1 into third_party/boost
+
+    https://www.boost.org/users/history
+    $ `wget https://dl.bintray.com/boostorg/release/1.65.1/source/boost_1_65_1.tar.gz`
+    $ `tar -xzf boost_1_65_1.tar.gz`
+    $ `cd boost_1_65_1`
+    
+    Link boost to SSF from boost_1_65_1 directory
```

Comment:
I had to guess where to do this. I did it from the ssf directory I got unarchiving the ssf sources above.